### PR TITLE
Add important =, dry out.

### PR DESCRIPTION
**Pull request by e0d**
@feanil @jarv currently this is busted.

### CODE REVIEW

File: util/jenkins/change-enrollment.sh [hunk 1]
```diff
@@ -3,10 +3,11 @@ pip install -r requirements.txt
 env
 
 ansible="ansible first_in_tag_Name_${environment}-${deployment}-worker -i playbooks/ec2.py -u ubuntu -s -U www-data -a"
-manage="/edx/bin/python.edxapp /edx/bin/manage.edxapp lms change_enrollment --settings aws"
+manage="/edx/bin/python.edxapp /edx/bin/manage.edxapp lms change_enrollment --settings=aws"
+noop=""
```
Comment (jarv) [inline util/jenkins/change-enrollment.sh]: is it correct to be setting the variable noop right before you are testing whether it is set to true?

Comment (e0d) [inline util/jenkins/change-enrollment.sh]: No, var name clash, noop is passed from the Jenkins job.

**Comment by jarv** [comment]
:+1:

**Comment by feanil** [comment]
:+1:

**Pull request merged**